src/App.tsx

```diff
@@ -1,61 +1,111 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
```

Comment:
The `usersFromServer` and `todosFromServer` imports should be handled in the `App` component, and the data should be passed down as props to child components. This aligns with the task requirements to prepare data in the `App` component.